tests/cpu_cortexm_address_check/Makefile

```diff
@@ -0,0 +1,10 @@
+FEATURES_REQUIRED += cpu_check_address
+
+BOARD ?= nucleo-l152re
+
+USEMODULE += shell
+USEMODULE += shell_commands
+
+include ../Makefile.tests_common
```

Comment:
usually tests include this first. the board selection must go above that, though.

Is there any reason not to use samr21-xpro, which is usually the first choice if native won't work, but there are no other restrictions?